app/controllers/reviews_controller.rb

```diff
@@ -26,12 +28,24 @@ def create
   def update
     @review = Review.find(params[:id])
     @review.update(review_params)
+    @airline = Airline.find(@review.airline_id)
     if @review.save
       flash[:notice] = 'Review successfully edited'
-      redirect_to airline_path
+      redirect_to airline_path(@airline)
     else
       flash[:notice] = 'Rating can\'t be blank'
-      render :new
+      redirect_to airline_review_path(airline_id: @airline.id, id: @review.id)
+    end
+  end
+  def destroy
+    @review = Review.find(params[:id])
+    @airline = Airline.find(@review.airline_id)
```

Comment:
You should find the airline and then find the review associatd with the airline like this:
@ariline.reviews.fin(params[:id])

That way its gauranteed that the review is associated with an airline